config-connector/solutions/iam/kpt/storage-bucket-iam/README.md

```diff
@@ -0,0 +1,36 @@
+Storage Bucket IAM
+==================================================
+# NAME
+  storage-bucket-iam
+# SYNOPSIS
+  Config Connector compatible yaml to enable permissions for a storage bucket.
+# REQUIREMENTS
```

Comment:
Could you convert the requirements from a paragraph into a list of bullet points to make them more noticeable?

Reply:
Mark the comment as blocking merge.